src/Day.js

```diff
@@ -150,7 +149,8 @@ export default class Day extends Component {
         onTouchStart={handleEvent(onTouchStart, day, modifiers)}
         onFocus={handleEvent(onFocus, day, modifiers)}
       >
-        {children}
+        <span className="DayPicker-ScreenReaderOnly">{ariaLabel}</span>
+        <span aria-hidden="true">{children}</span>
```

Comment:
I fear this line will be a breaking change for many. Could we just add `aria-live="polite"` and the "screen-reader-only part – or are these changes related?

Reply:
hey, friend! I'm one of Hannah's coworkers helping get this PR up to snuff. Is the issue that `Day`'s `children` would be pushed down a level in the DOM tree? We placed `children` outside of the span tag again with no poor effects from the screenreader's perspective, if that's acceptable.

Reply:
My problem is that I cannot add breaking changes to v7, as i have no time left :)

Could you open an issue describing better the problem?

My concern is that this change may break other DayPicker implementations: e.g. what happens to localized calendars? Must the existing DayPickers add an `ariaLabel` after this change?